### PR TITLE
feat(Icon): add new icons

### PR DESCRIPTION
**Pull request by chaitanyadeorukhkar**

### CODE REVIEW

File: src/icons/LifeBuoy/LifeBuoy.native.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```
Comment (kamleshchandnani) [inline src/icons/LifeBuoy/LifeBuoy.native.js:1]: LifeBuoy?? 🤣

Comment (chaitanyadeorukhkar) [reply src/icons/LifeBuoy/LifeBuoy.native.js:1]: Picked it from the figma file. Didn't know what else to call it 😅

File: src/icons/Settings/Settings.native.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```
Comment (kamleshchandnani) [inline src/icons/Settings/Settings.native.js:1]: Shouldn't this be called `Gear` Icon ?

Comment (chaitanyadeorukhkar) [reply src/icons/Settings/Settings.native.js:1]: It's `settings` on figma. Even I was confused. Should I change it to gear?

**Pull request merged**